app/controllers/mentor/messages_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Mentor::MessagesController < ApplicationController
+  def index
+    @messages = Message.where(chat_group_params)
+  end
+
+  def create
+    Message.create()
```

Comment:
__[RuboCop]__

Style/MethodCallWithoutArgsParentheses: Do not use parentheses for method calls with no arguments.


<!-- Comment by [SideCI](https://sideci.com) -->